### PR TITLE
[Help Doc] Added help file for instrument data entry

### LINKED ISSUE #5691: [Raisinbread] No data for help editor module?
It doesn't seem to be possible to edit any help files using the Raisinbread dataset. The Test VM, demo, and my sandbox contain no data.

How is this module being tested?

### LINKED ISSUE #6906: [Help Doc] No help document for the instruments module
There is no help document for the instruments module. 

When you navigate to a data entry page for a specific timepoint's instrument and click on the `?` button, there is a `No help available for instruments` message:
Screen Shot 2020-08-12 at 6 00 57 PM

### PR DESCRIPTION
**Pull request by AlexandraLivadas**
I added a markdown file under `modules/instruments/help/` so that Help instructions for the user will render when they are completing instrument data entry. 

Here is a screenshot of how the markdown renders in the frontend. 
Screen Shot 2020-08-12 at 2 43 56 PM

#### Testing instructions (if applicable)
1. Checkout this branch and go to the LORIS frontend.
2. Navigate to `Candidate`->`Access Profile` and choose any candidate with associated instruments.
3. Navigate to the Data Entry page for any of the timepoint's instruments.
4. Go to the top of the page and click on the `?`
5. View the markdown.

#### Link to related issue

* Resolves #6906

### CODE REVIEW

**Comment by AlexandraLivadas** [comment]
@christinerogers I made a note in the PR description about how this resolves #5691. I will take another crack at adding Help text specifically for the BMI instrument later today!

**Review by pierre-p-s — APPROVED** [review]

**Review by christinerogers — CHANGES_REQUESTED** [review]
Hey @AlexandraLivadas  - looking at the purpose of this help text, 
I think this can be condensed and simplified greatly (something we started with Help text with Alex Knoll, written up in her Help text guide)  : 
* see my 3 major edits
* delete everything below that - I don't think the content for each sample instrument is distinct enough to warrant the extra scrolling / re-reading. 

Might you have time to do these easy changes this week?  let us know...

File: modules/instruments/help/instruments.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Instrument
+
+
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:3]:
```suggestion
# Instruments

```

File: modules/instruments/help/instruments.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Radiology Review
+
+The Radiology Review instrument has one page of data entry. The required information is marked with a red asterisk.
+
+## AOSI
+
+The AOSI instrument has three pages of data entry. The required information is marked with a red asterisk.
+
+Along with the Administration and Data Entry options, the sidebar for this instrument includes the option to specify the Validity of the instrument. If the Data Entry status is set to `Complete`, the Validity option cannot be edited. If the Adminisitration or Validity options are not set by the user, the Data Entry status cannot be changed to `Complete`.
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:34]:
```suggestion
```

File: modules/instruments/help/instruments.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Instruments
+
+Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).
+
+In the left sidebar panel (on tablets, click the list icon) - you can set status flags for this instrument form.  The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set.
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:5]:
```suggestion
Status flags can be set in the left sidebar - the Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 
```

File: modules/instruments/help/instruments.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Instruments
+
+Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).
+
+In the left sidebar panel (on tablets, click the list icon) - you can set status flags for this instrument form.  The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 
+If a form has multiple pages, enter and save data on each subpage before returning to the Top page to set the status flags. 
+
+In the sidebar, the Data Entry status can be set to either `Complete` or `In Progress`. The status cannot be set to `Complete` if the Administration option is not set. If the Data Entry status is `In Progress`, the sidebar will display the option to delete all the instrument data. The sidebar also allows the user to specify the Administration status of the instrument.
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:8]:
```suggestion
The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set.
While the Data Entry status is `In Progress`, you may see in the sidebar the option to delete all the instrument data which will clear all data in the form. 
```

File: modules/instruments/help/instruments.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).
+
+In the left sidebar panel (on tablets, click the list icon) - you can set status flags for this instrument form.  The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 
+The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set.
+While the Data Entry status is `In Progress`, you may see in the sidebar the option to delete all the instrument data which will clear all data in the form. 
+
+Under the general information about the candidate at the top of the screen, the user can perform data entry for the instrument. Required values are specified with a red asterisk. If these values are not completed, an error message will appear when a user attempts to save the data. 
+
+If the instrument has multiple pages, the user must save the data individually for each page. If the user has navigated to a page that is not the `Top` page of the instrument, they will be unable to alter any status flags in the sidebar.
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:11]:
```suggestion
```
removing

File: modules/instruments/help/instruments.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Instruments
+
+Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).
+
+
```
Comment (christinerogers) [inline modules/instruments/help/instruments.md:5]:
```suggestion

```
remove extra space

**Comment by christinerogers** [comment]
Looks like @AlexandraLivadas hasn't had time for this in the past few weeks. 
I committed directly the last changes requested.  

@driusan are you comfortable doing final review?

**Review by christinerogers — APPROVED** [review]

**Comment by christinerogers** [comment]
@AlexandraLivadas if you have time still could you pull, update and re-post the screenshot, showing how the current text renders?  thanks!

**Comment by christinerogers** [comment]
@driusan final review please

**Comment by driusan** [comment]
this has not had travis run on it.

**Comment by laemtl** [comment]
@driusan The only change is a new markdown file, so I don't see any reason for Travis to fail. I closed/reopen to see if that can help, and can investigate further if the status does not update.

**Pull request merged**